pkg/metrics/grafana/tidb.json

```diff
@@ -22885,7 +22885,7 @@
           "targets": [
             {
               "exemplar": true,
-              "expr": "avg(tidb_server_ttl_watermark_delay{k8s_cluster=\"$k8s_cluster\",tidb_cluster=\"$tidb_cluster\", type=\"schedule\"}) by (type, name)",
+              "expr": "max(tidb_server_ttl_watermark_delay{k8s_cluster=\"$k8s_cluster\",tidb_cluster=\"$tidb_cluster\", type=\"schedule\"}) by (type, name)",
```

Comment:
Is this because the `delay` is reported by 1 node only?

Reply:
Yes, when a node is not a leader any more, we'll delete all related metrics on it.